components/org.wso2.carbon.identity.scim2.common/src/test/resources/dbscripts/identity.sql

```diff
@@ -0,0 +1,1282 @@
+);
+
+CREATE TABLE IF NOT EXISTS IDN_OAUTH2_AUTHZ_CODE_SCOPE(
+            CODE_ID   VARCHAR(255),
+            SCOPE     VARCHAR(60),
+            TENANT_ID INTEGER DEFAULT -1,
+            PRIMARY KEY (CODE_ID, SCOPE),
+            FOREIGN KEY (CODE_ID) REFERENCES IDN_OAUTH2_AUTHORIZATION_CODE (CODE_ID) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS IDN_OAUTH2_DEVICE_FLOW (
+            CODE_ID VARCHAR(255),
+            DEVICE_CODE VARCHAR(255),
+            USER_CODE VARCHAR(25),
+            CONSUMER_KEY_ID INTEGER,
+            LAST_POLL_TIME TIMESTAMP NOT NULL,
+            EXPIRY_TIME TIMESTAMP NOT NULL,
+            TIME_CREATED TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+            POLL_TIME BIGINT,
+            STATUS VARCHAR (25) DEFAULT 'PENDING',
+            AUTHZ_USER VARCHAR (100),
+            TENANT_ID INTEGER,
+            USER_DOMAIN VARCHAR(50),
+            IDP_ID INTEGER,
+            SUBJECT_IDENTIFIER VARCHAR(255),
+            PRIMARY KEY (DEVICE_CODE),
+            UNIQUE (CODE_ID),
+            UNIQUE (USER_CODE),
+            FOREIGN KEY (CONSUMER_KEY_ID) REFERENCES IDN_OAUTH_CONSUMER_APPS(ID) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS IDN_OAUTH2_DEVICE_FLOW_SCOPES (
+            ID INTEGER NOT NULL AUTO_INCREMENT,
+            SCOPE_ID VARCHAR(255),
+            SCOPE VARCHAR(255),
+            PRIMARY KEY (ID),
+            FOREIGN KEY (SCOPE_ID) REFERENCES IDN_OAUTH2_DEVICE_FLOW(CODE_ID) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS IDN_OAUTH2_ACCESS_TOKEN_SCOPE (
+            TOKEN_ID VARCHAR (255),
+            TOKEN_SCOPE VARCHAR (60),
+            TENANT_ID INTEGER DEFAULT -1,
+            PRIMARY KEY (TOKEN_ID, TOKEN_SCOPE),
+            FOREIGN KEY (TOKEN_ID) REFERENCES IDN_OAUTH2_ACCESS_TOKEN(TOKEN_ID) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS IDN_OAUTH2_SCOPE (
+            SCOPE_ID INTEGER NOT NULL AUTO_INCREMENT,
+            NAME VARCHAR(255) NOT NULL,
+            DISPLAY_NAME VARCHAR(255) NOT NULL,
+            DESCRIPTION VARCHAR(512),
+            TENANT_ID INTEGER NOT NULL DEFAULT -1,
+            SCOPE_TYPE VARCHAR(255) NOT NULL,
+            PRIMARY KEY (SCOPE_ID),
+            UNIQUE (NAME, TENANT_ID)
+);
+
+CREATE TABLE IF NOT EXISTS IDN_OAUTH2_SCOPE_BINDING (
+            SCOPE_ID INTEGER NOT NULL,
+            SCOPE_BINDING VARCHAR(255) NOT NULL,
+            BINDING_TYPE  VARCHAR(255) NOT NULL,
+            FOREIGN KEY (SCOPE_ID) REFERENCES IDN_OAUTH2_SCOPE(SCOPE_ID) ON DELETE CASCADE,
+            UNIQUE (SCOPE_ID, SCOPE_BINDING, BINDING_TYPE)
+);
+
+CREATE TABLE IF NOT EXISTS IDN_OAUTH2_RESOURCE_SCOPE (
+            RESOURCE_PATH VARCHAR(255) NOT NULL,
+            SCOPE_ID INTEGER NOT NULL,
+            TENANT_ID INTEGER DEFAULT -1,
+            PRIMARY KEY (RESOURCE_PATH),
+            FOREIGN KEY (SCOPE_ID) REFERENCES IDN_OAUTH2_SCOPE (SCOPE_ID) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS IDN_SCIM_GROUP (
```

Comment:
We can have this table only I suppose.

Reply:
addressed by https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/619/commits/1f437bab501e8b62d33978cc6288ba3a6ff7bd50